build/cluster.js

```diff
@@ -0,0 +1,92 @@
+// Copyright 2015 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Gulp tasks for kubernetes cluster management.
+ */
+import gulp from 'gulp';
+import chmod from 'gulp-chmod';
+import gunzip from 'gulp-gunzip';
+import untar from 'gulp-untar';
+import request from 'request';
+import source from 'vinyl-source-stream';
+import conf from './conf';
+
+
+let downScript = `${conf.paths.kubernates}/cluster/kube-down.sh`,
+    exec = require('child_process').exec,
+    file = 'kubernetes.tar.gz',
+    stableVersion = 'v1.0.7',
+    upScript = `${conf.paths.kubernates}/cluster/kube-up.sh`;
+
+
+/**
+ * Creates cluster from scratch.
+ * Downloads latest version of kubernetes.
+ * Execute script to up cluster.
+ *
+ */
+gulp.task('create_cluster', ['download_kubernete_release'], function (done) {
+    exec(`KUBERNETES_PROVIDER=vagrant VAGRANT_DEFAULT_PROVIDER=libvirt ${upScript}`,
+        function (err, stdout) {
+            if (err)
+                return console.error(err);
+            console.log(stdout);
+            return done();
+        });
+});
+
+
+/**
+ * Downloads latest version of kubernetes.
+ */
+gulp.task('download_kubernete_release', function () {
+    let releaseUrl =
+        `https://storage.googleapis.com/kubernetes-release/release/${stableVersion}/${file}`;
+    console.log(releaseUrl);
+    return request(releaseUrl)
+        .pipe(source(file))
+        .pipe(gunzip())
+        .pipe(untar())
+        .pipe(chmod(755))
+        .pipe(gulp.dest(conf.paths.tmp));
+});
+
+
+/**
+ * Bring up a Kubernetes cluster.
+ */
+gulp.task('up_cluster', function (done) {
```

Comment:
Please use doneFn (or fooFn in general) for variables/parameters that are functions. Other parts of the code have been following this pattern. Again, this will be written in a style guide, once we have time to do this.